libraries/core/src/test/java/com/paritytrading/philadelphia/FIXInitiatorTest.java

```diff
@@ -408,18 +410,22 @@ void sendApplicationMessage() throws IOException {
         initiatorMessages(messages);
     }
 
```

Comment:
hi, this is the only part where the codebase relies on blocking behaviour.
Is it still wanted / needed?